splink/athena/athena_linker.py

```diff
@@ -70,14 +69,12 @@ def validate(self):
         pass
 
     def drop_table_from_database(self, force_non_splink_table=False):
-
         self._check_drop_folder_created_by_splink(force_non_splink_table)
         self._check_drop_table_created_by_splink(force_non_splink_table)
         self.athena_linker.drop_table_from_database_if_exists(self.physical_name)
         self.athena_linker.delete_table_from_s3(self.physical_name)
 
     def _check_drop_folder_created_by_splink(self, force_non_splink_table=False):
```

Comment:
Oops it looks like my black configuration is slightly different and deleted some lines. Want me to revert these changes?

Reply:
No need, it's fine.